symphony-bdk-core-invokers/symphony-bdk-core-invoker-api/src/main/java/com/symphony/bdk/core/api/invoker/util/ApiUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.symphony.bdk.core.api.invoker.util;
+
+import org.apiguardian.api.API;
+
+@API(status = API.Status.INTERNAL)
+public class ApiUtils {
+
+  /**
+   * Creates a user agent string used for the User-Agent header
+   *
+   * @return a user agent string containing the current BDK version
+   */
+  public static String getUserAgent() {
+    String jarVersion = ApiUtils.class.getPackage().getImplementationVersion();
+    String bdkVersion = jarVersion == null ? "2.0" : jarVersion;
+
+    return "Symphony BDK/" + bdkVersion + "/java/1.8";
```

Comment:
looks like we can retrieve the Java version with https://stackoverflow.com/questions/2591083/getting-java-version-at-runtime

Usually utility classes are made final and with a private constructor to avoid instantiation since they only have static methods

Reply:
👍